### PR TITLE
Make experiment details page responsive (closes #1251).

### LINKED ISSUE #1251: Make Experiment Details Pages Fully Responsive
Now that we've made these things visible without logging it, we're going to need them to be responsive.

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @johngruen @lmorchard 
## Screenshots

With add-on: [360px](https://cloud.githubusercontent.com/assets/23885/17941650/798be752-69f1-11e6-8004-5b7741bd0daa.png), [768px](https://cloud.githubusercontent.com/assets/23885/17941655/8526f660-69f1-11e6-98d8-f910d2b6d4ba.png)
Without add-on: [360px](https://cloud.githubusercontent.com/assets/23885/17941664/90fd5ede-69f1-11e6-83cb-11b9d5188a45.png), [768px](https://cloud.githubusercontent.com/assets/23885/17941675/979d870a-69f1-11e6-8cb6-b13c2063440a.png)
Non-Firefox: [360px](https://cloud.githubusercontent.com/assets/23885/17941788/07550cd0-69f2-11e6-9273-1a760a626275.png), [768px](https://cloud.githubusercontent.com/assets/23885/17941794/0e964464-69f2-11e6-9ad8-92765ec6b6da.png) (everything else is the same as the no-addon shots)

### CODE REVIEW

File: testpilot/frontend/static-src/app/views/testpilot-promo-view.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import installAddon from '../lib/install-addon';
 export default BaseView.extend({
   _template: `
     <div class="experiment-promo">
-      <div class="reverse-split-banner responsive-git content-wrapper">
+      <div class="reverse-split-banner responsive-content-wrapper">
```
Comment (chuckharmston) [inline testpilot/frontend/static-src/app/views/testpilot-promo-view.js:8]: This was me having focus in the wrong tmux pane (－‸ლ)

**Comment by chuckharmston** [comment]
Some updates now reflected in the PR:

New caption line height:

screen shot 2016-08-24 at 12 15 11 pm

Font size and margins of the "Try out..." header on small screens:

screen shot 2016-08-24 at 12 19 30 pm

Footer spacing with add-on installed:

screen shot 2016-08-24 at 12 22 15 pm

**Comment by johngruen** [comment]
#1251

**Comment by lmorchard** [comment]
Works for me! Going to merge and we can iterate further after folks get a look at dev, if necessary

**Pull request merged**